plugins/source/azure/resources/services/security/regulatory_compliance_standards.go

```diff
@@ -44,9 +46,23 @@ func fetchRegulatoryComplianceStandards(ctx context.Context, meta schema.ClientM
 	for pager.More() {
 		p, err := pager.NextPage(ctx)
 		if err != nil {
-			return err
+			return checkNoStandardPricingBundle(err)
```

Comment:
I'm not 100% sure when this situation occurs, but perhaps we should log this as an `info`/`warn` message, so users can be aware?

Reply:
It can happen if there's no std pricing bundle for that subscription, it just should be ignored.

Reply:
Why should it be ignored and not logged? What does the pricing bundle have to do with regulatory compliance standards? Like I said, maybe I don't understand, but I'd think the user may want to add a standard pricing bundle to the subscription so they can get this table.

Reply:
I've added the warn log, but the message text is pretty clear, see #7197

Reply:
Thanks for adding the log. I saw the message and agree it's useful, but users would never see it if we didn't log it.